packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -672,18 +673,19 @@ export class ModularChangeFamily
 			change.change.destroys === undefined,
 			0x89a /* Unexpected destroys in change to invert */,
 		);
+		const { revInfos } = getRevInfoFromTaggedChanges([change]);
 
 		if (hasConflicts(change.change)) {
 			return makeModularChangeset(
-				undefined,
-				undefined,
-				undefined,
-				undefined,
-				undefined,
+				undefined /* fieldChanges */,
+				undefined /* nodeChanges */,
+				undefined /* nodeToParent */,
+				undefined /* nodeAliases */,
+				undefined /* crossFieldKeys */,
 				change.change.maxId,
-				[],
-				undefined,
-				undefined,
+				revInfos,
```

Comment:
This `revInfos` reprents information about the revisions that make up the change that is being inverted. That the inverse change being created here should not have the same information since it is a different change.
I think `invert` needs to take in a new RevisionTag parameter and construct a new `RevInfo` based on that. You can make the new revision parameter optional for now.

Reply:
Updated it. Please take a look.